ddtrace/appsec/_handlers.py

```diff
@@ -194,22 +194,22 @@ def _on_request_init(wrapped, instance, args, kwargs):
     if _is_iast_enabled():
         try:
             from ddtrace.appsec._iast._taint_tracking import OriginType
+            from ddtrace.appsec._iast._taint_tracking import origin_to_str
             from ddtrace.appsec._iast._taint_tracking import taint_pyobject
             from ddtrace.appsec._iast.processor import AppSecIastSpanProcessor
 
             if not AppSecIastSpanProcessor.is_span_analyzed():
                 return
 
-            # TODO: instance.query_string = ??
             instance.query_string = taint_pyobject(
                 pyobject=instance.query_string,
-                source_name=OriginType.QUERY,
+                source_name=origin_to_str(OriginType.QUERY),
```

Comment:
this `origin_to_str` should probably be done on the `taint_pyobject` function when getting an origin as `source_name` so we don't have to do it on each call to `taint_pyobject`.

Reply:
source name is the key value (that is, a string) in most of the cases, but if we don't have any key we set the origin type